src/Serilog.Sinks.Raygun/Serilog.Sinks.Raygun.csproj

```diff
@@ -31,6 +31,7 @@
   </ItemGroup>
 
   <ItemGroup>	
+	<PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="1.1.2" />	
```

Comment:
Should this be moved to the netstandard2.0 group above?

Reply:
Sure, I'll move it.